packages/fuels-core/src/lib.rs

```diff
@@ -107,6 +111,7 @@ impl fmt::Display for ParamType {
 #[derive(Debug, Clone, PartialEq, EnumString)]
 #[strum(ascii_case_insensitive)]
 pub enum Token {
+    Unit,
```

Comment:
How about adding a comment here explaining `Unit`? If you have no context, `Unit`, both the name and the purpose, can be very confusing. E.g., We use `Unit` when the type being handled is an "empty" `Enum`, so why `Unit` and not `Enum` with an empty `Box<>`? Also maybe hint that the string type coming from the compiler is `"()"`, etc.

Reply:
Yes you're right!